third-party/rsutils/include/rsutils/py/pybind11.h

```diff
@@ -52,11 +52,11 @@ using namespace pybind11::literals;
     }
 #define FN_FWD( CLS, FN_NAME, PY_ARGS, FN_ARGS, CODE )                                                                 \
     #FN_NAME, []( CLS & self, std::function < void PY_ARGS > callback ) {                                              \
-        self.FN_NAME( [&self,callback] FN_ARGS { FN_FWD_CALL( CLS, FN_NAME, CODE ) } );                                \
+        self.FN_NAME( [self = &self,callback] FN_ARGS { FN_FWD_CALL( CLS, FN_NAME, CODE ) } );                         \
```

Comment:
What is the difference between &self and self =&self?
Don't you prefer a new name if you go on this path? to avoid confusion?

Reply:
`&self` seems to have been not working: C++ chose to **copy** `self`!
But some classes cannot be copied....
To ensure this doesn't happen, I chose to make `self` a pointer, kind of like `this` -- except cannot use `this`.